### PR TITLE
rgw: add documentation for upgrading with rgw_region_root_pool

### PR DESCRIPTION
**Pull request by oritwas**
Signed-off-by: Orit Wasserman <owasserm@redhat.com>

### CODE REVIEW

**Comment by cbodley** [comment]
don't we need some other page to link to this new one?

File: doc/radosgw/upgrade_to_jewel_with_different_region_pool.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+=============
+RGW upgrading to Jewel with a different rgw region root pool
+=============
+
+.. versionadded:: Jewel
+
+Upgrade of :term:`Ceph Object Gateway` to jewel with a different
+rgw region root pool. This document describes the needed configuration changes.
```
Comment (cbodley) [inline doc/radosgw/upgrade_to_jewel_with_different_region_pool.rst]: might want to be more specific about when this applies. we should also mention the settings for realm/period pools. something like this?

```markdown
If an existing multisite configuration uses a non-default setting for
`rgw region root pool`, the new pool settings `rgw zonegroup root pool`,
`rgw period root pool`, and `rgw realm root pool` should be set to match.
```

**Comment by theanalyst** [comment]
> don't we need some other page to link to this new one?
Maybe the multisite doc itself or one of the the config. reference pages

File: doc/radosgw/upgrade_to_jewel_with_different_region_pool.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Update configuration before upgrade
+================
+Add rgw_zonegroup_root_pool with the same pool name as rgw_region_root_pool.
```
Comment (theanalyst) [inline doc/radosgw/upgrade_to_jewel_with_different_region_pool.rst]: nit emphasize/use monospace for ``rgw region root pool``

**Pull request merged**